tools/linter_lib/custom_check.py

```diff
@@ -116,10 +116,6 @@
             "pattern": "msgid|MSGID",
             "description": 'Avoid using "msgid" as a variable name; use "message_id" instead.',
         },
-        {
-            "pattern": r".*blueslip.warning\(.*",
-            "description": "The module blueslip has no function warning, try using blueslip.warn",
-        },
```

Comment:
(ESLint catches this now, via [`import/namespace`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/namespace.md).)